tests/api/test_pilot_manager.py

```diff
@@ -19,14 +18,14 @@
 if DBURL is None:
     print "ERROR: RADICAL_PILOT_DBURL (MongoDB server URL) is not defined."
     sys.exit(1)
-    
-DBNAME = os.getenv("RADICAL_PILOT_TEST_DBNAME")
+
+DBNAME = os.getenv("RADICAL_PILOT_TEST_DBNAME", 'test')
```

Comment:
Apparrently, MLab does not work the way it used anymore. Also, I do not see where the DBURL is set. Is it fair to assume that this is taken care of?

Reply:
Out of scope for this PR.  Yes, we need to find a MongoDB servcice replacement... :/

Reply:
Yes, URL setting should be taken case of.